core_v14/src/lib.rs

```diff
@@ -21,3 +21,6 @@ pub mod value;
 pub use decoder::Decoder;
 pub use metadata::Metadata;
 pub use value::Value;
+
+/// A re-export of the [`scale_info`] crate, since we delegate much of the type inspection to it.
```

Comment:
What does `re-export` mean, as in where else is this getting exported? Trying to get more familiar with whats going on here :)

Reply:
It's common for libraries that manipulate data expressed in types stemming from another crate (like here, much of what is going on in `desub` relies on `scale-info` types) to "adopt" or "forward" that other crate's types as if they were its own. 
One instance of this is how the rust standard library re-exports types and code modules from the `core::` namespace as `std::`. The `core::` set of types and modules do not require a `std` environment but that's not very important to `std`-using applications, so for convenience the `core::*` modules are also available through `std::*`.
In this case James is re-exporting all of `scale-info`, allowing users to access them through the `desub` path, e.g. the `use desub::TypeInfo`, rather than `use scale_info::TypeInfo`. This is convenient and perfectly fine because it is not a feature of `desub` to have "swappable" scale type info backends.

Clear as mud? :)

Reply:
(Thank you! Nit: they'd have access via `desub::scale_info::TypeInfo` as it stands)